client/my-sites/plans-features-main/index.tsx

```diff
@@ -691,6 +692,15 @@ const PlansFeaturesMain = ( {
 		[]
 	);
 
+	const handlePlanTypeSelectorChange = useCallback( ( selectedItem: { url: string } ) => {
```

Comment:
`Note to self`:

If we go with this approach, it's probably worthwhile to add an inline comment describing that `url` can be a query string instead of a url